dev/com.ibm.ws.org.jboss.weld3/bnd.overrides

```diff
@@ -12,7 +12,9 @@ Specification-Title: JSR-365 Contexts and Dependency Injection for Java
 Specification-Version: 2.0
 DynamicImport-Package: *
 Export-Package: org.jboss.*;version="3.1.1.Final", \
+ !org.jboss.environment.se.*, \
```

Comment:
The parameters to `Export-Package` are processed in order. To get the behaviour you're looking for, you need to put the exclusion first.

For reference see https://bnd.bndtools.org/heads/private_package.html

As a side note, the current bundle manifest ends up exporting `org.jboss.weldx.transaction` twice, once at v3.1.1.Final and once at 1.0.16. The package does exist within the final bundle but contains no classes. I can't find this package in any of the source jars.